.github/workflows/stale.yml

```diff
@@ -0,0 +1,39 @@
+name: Stale check
+on:
+  workflow_dispatch: {}
+  schedule:
+    - cron: '0 0 * * *'
+permissions:
+  issues: write
+  pull-requests: write
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/stale@v6.0.1
+        with:
+          close-issue-reason: not_planned
+          days-before-stale: 60
+          days-before-close: 15
+          stale-issue-message: >
+            This issue has been automatically marked as stale because it has
+            not had any activity in the past 60 days.
+
+            The next time this stale check runs, the stale label will be
+            removed if there is new activity. The issue will be closed after 15
+            days if there is no new activity. Apply keepalive label to exempt this Issue.
+
+            Thank you for your contributions!
```

Comment:
I like this message except for this line (and same below). It just feels a little hollow for the stalebot to be thanking people as it's auto closing issues and PRs.

Other than that, it's clear what's happening and how to avoid it. I'm good on this.

Reply:
done, updated